apps/dashboard/src/lib/components/Question/RadioQuestion/index.svelte

```diff
@@ -80,25 +98,77 @@
     <CodeSnippet {code} />
   {/if}
 
-  <div class={!disableOptContainerMargin && 'ml-4'}>
-    {#each options as option}
-      <button
-        class="cursor-pointer text-left my-2 border-2 border-gray-300 rounded-md hover:bg-gray-200 dark:hover:bg-neutral-800 w-full {getValidationClassName(
-          option
-        )}"
-        type="button"
-      >
-        <RadioItem
-          className="p-2"
-          {name}
-          value={option.value}
-          checked={defaultValue.includes(option.value) && option.is_correct}
-          label={option.label || option.value}
-          {disabled}
-        />
-      </button>
-    {/each}
-  </div>
+  {#if !gradeWithAI}
+    <div class={!disableOptContainerMargin && 'ml-4'}>
+      {#each options as option}
+        <button
+          class="cursor-pointer text-left my-2 border-2 border-gray-300 rounded-md hover:bg-gray-200 dark:hover:bg-neutral-800 w-full {getValidationClassName(
+            option
+          )}"
+          type="button"
+        >
+          <RadioItem
+            className="p-2"
+            {name}
+            value={option.value}
+            checked={defaultValue.includes(option.value) && option.is_correct}
+            label={option.label || option.value}
+            {disabled}
+          />
+        </button>
+      {/each}
+    </div>
+  {:else}
+    <div class="ml-4 border rounded-md">
+      <div>
+        {#each options as option}
+          <button
+            class="cursor-pointer text-left mb-4 border-2 border-gray-300 rounded-md hover:bg-gray-200 dark:hover:bg-neutral-800 w-full {getValidationClassName(
+              option
+            )}"
+            type="button"
+          >
+            <RadioItem
+              className="p-2"
+              {name}
+              value={option.value}
+              checked={defaultValue.includes(option.value) && option.is_correct}
+              label={option.label || option.value}
+              {disabled}
+            />
+          </button>
+        {/each}
+      </div>
+      {#if isLoading}
+        <GradeLoader />
+      {:else}
+        <div class="flex items-start justify-between px-2 py-4">
+          <div class="flex items-center space-x-4">
+            <img src="/ai.svg" alt="alt" />
+            <p class="font-normal text-sm">
+              {reason}
+            </p>
+          </div>
+          <div class="flex space-x-2">
+            <PrimaryButton
+              variant={VARIANTS.CONTAINED_SUCCESS}
+              label="Accept"
+              className="rounded-none py-1 px-2"
+              disablePadding={true}
+              onClick={acceptGrade}
+            />
+            <PrimaryButton
+              variant={VARIANTS.CONTAINED_DANGER}
+              label="Reject"
+              className="rounded-none py-1 px-2"
```

Comment:
@Chifez I noticed you duplicated this logic in the 3 question component.

Can you please move it into a reusable component and it should be used in each of the question components.